### ISSUE TITLE
Add support for Chat device

### ISSUE THREAD

**Issue by pannal** [inoahguy02/maxwell-cutoff-fix#6]
Hey,

Thank you for this, it's working great.

One thing I've noticed: It only keeps the Default/Game device active. The Chat device has the same old issue. Would it be possible to keep that one alive, as well?

Thanks!

**Comment by inoahguy02** [comment]
I found that `rodio` and `cpal` crates solve this:

```
use cpal::traits::{DeviceTrait, HostTrait}

let host = cpal::default_host();

let device = host.devices().find(|device| {
    device.name().unwrap().contains("Audeze Chat or som")
}).unwrap();

let manual_stream = rodio::OutputStream::try_from_device(device).unwrap();
loop { /*sleepforever*/ };
```

It got rid of the cutoff when I used `OutputStream::try_default()`, but I did not test other devices and don't know if it still allows auto-shutoff

Two ways to go about this:
1. Have a config that contains sound devices that should be kept alive
2. Keep all sound devices alive

I'd like to go with 1, would be cool to have a CLI that the user can interact with to see name of devices and add/remove them

Device names can be displayed by using this:

```
for (index, device) in host.devices().unwrap().enumerate() {
    println!("Device {}: {}", index, device.name().unwrap());
}
```

**Comment by inoahguy02** [comment]
Made proposed fix in https://github.com/inoahguy02/maxwell-cutoff-fix/commit/b4b1c84f9473654118537a9d03dd4c7f457f18d2. Needs testing

**Comment by inoahguy02** [comment]
Added both devices to config and tested. Seems to work for me
